io/plugins/eu.esdihumboldt.hale.io.json.test/src/eu/esdihumboldt/hale/io/json/test/JsonInstanceCollectionTest.groovy

```diff
@@ -58,29 +60,29 @@ class JsonInstanceCollectionTest {
   "features": [
     {
       "type": "Feature",
-      "geometry": {
-        "type": "LineString",
-        "coordinates": [
-          [102.0, 0.0], [103.0, 1.0], [104.0, 0.0], [105.0, 1.0]
-        ]
-      },
       "properties": {
         "id": "line",
-        "name": "Line feature"
+        "name": "Line feature",
+		"geometry": {
```

Comment:
In principle there can be a geometry inside a property, but in GeoJson the geometry is primarily inside the `geometry` field next to the `properties`. Please have a look at the GeoJson spec.